### DISCUSSION TITLE
Ver5.9/6.0 "TableToArray" behave differently: 「&nbsP;」 no longer regarded as Space but「？」

### DISCUSSION THREAD

**Discussion by hanamichi77777** [GCuser99/SeleniumVBA#138] [Q&A] [answered]
Hello,I always use SeleniumVBA. Thank you very much.
I have a favor to ask of you.
「&nbsP;」 is often used.　※convert 「p」 to uppercase
Please, is it possible to repair ?

**Example:**
**＜HTML＞**
![タイトルなし](https://github.com/user-attachments/assets/687a63f5-4f3c-4c0b-968f-bbeb47559afc)
**＜OUTPUT＞**
![タイトルなし3](https://github.com/user-attachments/assets/6a681a2c-a9b4-4e0c-ab8a-a028b644a061)

**Ver5.8 and before:**
The content you get via TableToArray is "Hi, this is \r\n  \r\n  Mike"

**Ver5.9/6.0**
The content you get via TableToArray is "Hi, this is ?\r\n  \r\n  Mike"
※「&lT;」「&gT;」「&amP;」「&quoT;」is working properly. Only 「&nbsP;」 is not  working

**Comment by GCuser99** [comment]
Thanks for reporting this @hanamichi77777 - will be fixed in next update.

**Reply by hanamichi77777** [reply]
Thank you very much.
When I searched for the character code with GetText, it was number 32. 
From this result, it seems that it is considered the same as a half-width space.

**＜HTML＞**
![タイトルなし](https://github.com/user-attachments/assets/92ce79eb-a79f-4b3b-9ce5-7b0687f9b52b)

**＜Character Code＞※Get with GetText**
![タイトルなし3](https://github.com/user-attachments/assets/e2d7b261-11b7-4407-b979-e9d14026f3b5)

**Comment by GCuser99** [comment, marked as answer]
Fixed in v6.1

**Reply by hanamichi77777** [reply]
It was perfect. 
Thank you very much, [@GCuser99](https://github.com/GCuser99)
